src/less-style/modals.less

```diff
@@ -37,7 +37,7 @@
 }
 
 // Full-screen modal for displaying form XML
-.fd-source-modal {
+.fd-full-screen-modal {
```

Comment:
Doesn't this rename mean you can remove `fd-source-modal` from `core.js`? I don't see any other usages of `fd-source-modal`.